Jenkinsfile

```diff
@@ -3,19 +3,23 @@
 library 'kibana-pipeline-library'
 kibanaLibrary.load()
 
-kibanaPipeline(timeoutMinutes: 155, checkPrChanges: true, setCommitStatus: true) {
-  slackNotifications.onFailure(disabled: !params.NOTIFY_ON_FAILURE) {
-    githubPr.withDefaultPrComments {
-      ciStats.trackBuild {
-        catchError {
-          retryable.enable()
-          kibanaPipeline.allCiTasks()
-        }
-      }
-    }
-  }
-
-  if (params.NOTIFY_ON_FAILURE) {
-    kibanaPipeline.sendMail()
-  }
+kibanaPipeline(timeoutMinutes: 150) {
```

Comment:
I will revert all of the changes here as soon as I got the flaky test suite runner build ended. That change was suggested by @brianseeders as a way to test for a previous problem we had with memory resources on the ES Snapshot jobs.